homeworks/03-html-basics/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description"
+        content="Personal website about Maria Nanys. Picture of Maria Nanys and contact form with Maria Nanys.">
+    <meta name="twitter:card" content="summary">
+    <meta name="twitter:site" content="@mariananys" />
+    <meta name="twitter:creator" content="@mariananys" />    
+    <meta name="og:type" property="og:type" content="website" />
+    <meta name="og:url" property="og:url" content="https://www.mariananys.com/">
+    <meta name="og:title" property="og:title" content="Maria Nanys - Frontend Developer">
+    <meta name="og:image" property="og:image"
+        content="https://media.licdn.com/dms/image/D4D03AQEklI4U0Ai8Iw/profile-displayphoto-shrink_400_400/0/1694620754245?e=1718236800&v=beta&t=0SDNRx_A6QWVswWIfcqk_pBlkVfoV03y1Kk7RYF8CK4" />
+    <meta name="og:image:type" property="og:image:type" content="image/jpeg" />
+    <meta name="og:image:width" property="og:image:width" content="200" />
+    <meta name="og:image:height" property="og:image:height" content="200" />
+    <meta name="og:image:alt" property="og:image:alt" content="Smiling long-haired woman in front" />
+    <meta name="og:description" property="og:description"
+        content="Personal website about Maria Nanys. Picture of Maria Nanys and contact form with Maria Nanys.">
+    <title>Maria Nanys - Frontend Developer</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="index.html">Home</a>
+            <a href="pages/contact.html">Contact</a>
+        </nav>
+        <hr>
+    </header>
+    <main>
+        <h1>Hi, I am Maria Nanys,<br />Front-end Developer</h1>
+        <section>
```

Comment:
I think section is redundant here, because p describes all the sense of content.